readme.md

```diff
@@ -25,7 +37,8 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```
+You'll need to attach a submit handler to the form that prevents the default action from being carried out, and instead directs it to an AJAX request that is built like the code above, except with 'type: "POST"' instead of "GET". Each field of the form will need to be appropriately mapped to a column in the database (I think this is with name="whatever" in the HTML tag for the field ...)
 
```

Comment:
The form is mapped to a object using the `data:` property in the AJAx request. This may correspond to structures in a database, but not necessarily.